src/components/molecules/SectionRenderer/ItemRenderer/ItemRenderer.tsx

```diff
@@ -70,81 +72,85 @@ function ItemRenderer<ItemType, ScopeType>(props: ItemRendererProps<ItemType, Sc
     scrollContainer.current?.scrollIntoView();
   }, [checkedResourceIds.length, itemInstance.shouldScrollIntoView]);
 
+  const Wrapper = ContextMenuWrapper.Component ? ContextMenuWrapper.Component : 'div';
```

Comment:
Could this be written in a `useMemo` ? Maybe `ContextMenuWrapper.Component` could return `div` as default from useItemCustomization?

Reply:
Actually I was thinking about that maybe some items must not have context menu option, so thats why I checked that and returned `div`. 
But, `KustomizationSectionBlueprint` is constant isn't it? So in every situtation that blueprint should have context menu (because every item has "Go to file" option in kustomize section). So I'm considering removing this line and rename `<Wrapper>` to `<ContextMenuWrapper.Component>`. What do you think about this one?

Sorry if I missed a point and please inform me about that.

Reply:
Some SectionBlueprints could definitely not have a ContextMenu, like the sections related to Helm or the parent sections in the Navigator, I think we could leave it like this for now